index.js

```diff
@@ -50,7 +50,7 @@ const deepMerge = (...sources) => {
 					value = deepMerge(returnValue[key], value);
 				}
 
-				returnValue = {...returnValue, [key]: value};
+				returnValue = { ...returnValue, [key]: value };
```

Comment:
Please don't make unrelated changes.

Reply:
Yeah, for sure, I forgot to disable prettier, will update